leakcanary-android/src/main/java/com/squareup/leakcanary/CanaryLog.java

```diff
@@ -0,0 +1,59 @@
+package com.squareup.leakcanary;
+
+import android.util.Log;
+
+public final class CanaryLog {
+
+  private static volatile Logger logger = new DefaultLogger();
+
+  public interface Logger {
+    void d(String message, Object... args);
+
+    void d(Throwable throwable, String message, Object... args);
```

Comment:
Maybe call it `e()` or `w()`? Because there are two kind of logs in the LeakCanary: "info" and "error", will be better to see problems as problems in the logcat :)